### PR TITLE
[AKS] FIX #12871: az aks enable-addons: Autogenerated help example is wrong for vitual-node option

### LINKED ISSUE #12871: Autogenerated help example is wrong for `az aks enable-addons`
### **This is autogenerated. Please review and update as needed.**

The example in the help message of this command is wrong, and since it's autogenerated, I'm not sure how to commit a fix.

**Command Name**
`az aks enable-addons`

**Errors:**
This is the example
> Examples
>     Enable Kubernetes addons. (autogenerated)
>         az aks enable-addons --addons virtual-node --name MyManagedCluster --resource-group
>         MyResourceGroup

This is further up the help message, describing that --subnet is a required parameter:
> virtual-node - enable AKS Virtual Node (PREVIEW). Requires --subnet-name to provide the
>         name of an existing subnet-name for the Virtual Node to use.

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az aks enable-addons --help`

## Expected Behavior

The example show that --subnet-name is required

## Environment Summary
```
Linux-4.19.104-microsoft-standard-x86_64-with-debian-buster-sid
Python 3.6.5

azure-cli 2.2.0 *

```

### PR DESCRIPTION
**Pull request by MikkelHegn**
Manual update of aks enable-addon command help text

**Description**  
Fixes autogenerated help message for aks enable-addon. #12871 

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
aks

**Comment by MikkelHegn** [comment]
Fixed PR title.

**Comment by yungezz** [comment]
hi @mirdaki This is an auto-generated AKS example, but missing a parameter. Could you pls help to look at from Aladdin side also? Goal is that next Aladdin offline pacakge will not refresh this fixing, and will not happen.

**Review by yungezz — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/acs/_help.py [hunk 1]
```diff
@@ -517,7 +517,7 @@
     short-summary: The resource ID of an existing Log Analytics Workspace to use for storing monitoring data.
 examples:
   - name: Enable Kubernetes addons. (autogenerated)
-    text: az aks enable-addons --addons virtual-node --name MyManagedCluster --resource-group MyResourceGroup
+    text: az aks enable-addons --addons virtual-node --name MyManagedCluster --resource-group MyResourceGroup --subnet MySubnetName
```
Comment (arrownj) [inline src/azure-cli/azure/cli/command_modules/acs/_help.py:520]: Why do we need to update this `autogenereated` example ?

Comment (MikkelHegn) [reply src/azure-cli/azure/cli/command_modules/acs/_help.py:520]: Because it doesn't contain all the required parameters. Please see #12871

**Comment by mirdaki** [comment]
@yungezz I've looped in the rest of the Aladdin team to make sure this is addressed.

File: src/azure-cli/azure/cli/command_modules/acs/_help.py (hunk 1 above)
Comment (arrownj) [reply src/azure-cli/azure/cli/command_modules/acs/_help.py:520]: Add @haroldrandom to review the Aladdin part.

**Comment by arrownj** [comment]
Hi @mirdaki , could you please update the PR title following  [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests)

**Review by haroldrandom — APPROVED** [review]

**Comment by MikkelHegn** [comment]
@arrownj - updated PR title

**Review by arrownj — APPROVED** [review]

**Pull request merged**